gooddata-pandas/gooddata_pandas/data_access.py

```diff
@@ -441,6 +444,9 @@ def compute_and_extract(
 
     response, col_to_attr_idx, col_to_metric_idx, index_to_attr_idx = result
 
+    if on_execution_submitted is not None:
+        on_execution_submitted(response)
```

Comment:
the `response` name is now not well-matching. would be better to do mild refactors?

Reply:
good point, changed!